mm2src/mm2_main/tests/docker_tests/swap_watcher_tests.rs

```diff
@@ -84,6 +84,11 @@ enum SwapFlow {
 
 #[allow(clippy::too_many_arguments)]
 fn start_swaps_and_get_balances(
+    /// Simulates an on-chain atomic swap between two parties (Alice and Bob) using HTLC principles.
+///
+/// The swap flow enforces that funds are locked in smart contracts and can only be redeemed
+/// by revealing a valid secret before the timelock expires, or refunded otherwise.
+/// This function is used in integration tests to verify swap logic, balances and watcher behavior.
```

Comment:
Why add this comment to this specific test, this test is in the watchers testing code and there are other tests that simulate a more basic swap.